features/config-create.feature

```diff
@@ -66,6 +66,30 @@ Feature: Create a wp-config file
       Error: The site you have requested is not installed
       """
 
+    Given a wp-config-extra.php file:
+      """
+      define( 'WP_DEBUG', true );
+      """
+
+    When I run `rm wp-custom-config.php`
+    Then the wp-custom-config.php file should not exist
```

Comment:
I would switch these around in order, so that we do the clean up of the previous configuration and then start with a new one afterwards.
```suggestion
    When I run `rm wp-custom-config.php`
    Then the wp-custom-config.php file should not exist

    Given a wp-config-extra.php file:
      """
      define( 'WP_DEBUG', true );
      """
```